main.go

```diff
@@ -22,18 +22,20 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"net"
 	"net/http"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 	"time"
 
-	"golang.org/x/sync/errgroup"
-
+	"github.com/QubitProducts/exporter_exporter/pkg/socks5"
```

Comment:
Please just vendor/ this and give appropriate attribute in a NOTICE flile

Reply:
It needed some changes.

Reply:
https://github.com/QubitProducts/exporter_exporter/pull/10/commits/d09ed2205dd357ecb88ffa1bcb8aa0dd5c4ae8aa

Reply:
That change is maybe not the real solution, just something to get it working for the POC.